tests/test-stake-pool-sdk.ts

```diff
@@ -143,197 +145,270 @@ describe("test basic functionalities", () => {
     }
   });
 
-  describe("testnet executions", () => {
-    let connection: Connection;
-    let stakerKeypair: Keypair;
-    let staker: MockWalletAdapter;
-    let originalBalanceLamports: number;
-    let scnSolMintPubkey: PublicKey;
-    let scnSolToken: Token;
-    let scnSolAtaPubkey: PublicKey;
-
-    before(async () => {
-      connection = new Connection(clusterApiUrl("testnet"));
-      // prep wallet and airdrop SOL if necessary
-      ({ stakerKeypair, staker, originalBalanceLamports } = await prepareStaker(
-        connection,
-      ));
-      console.log("staker:", staker.publicKey.toBase58());
-      console.log("original balance:", originalBalanceLamports);
-
-      const socean = new Socean("testnet");
-      const stakePool = await socean.getStakePoolAccount();
-      scnSolMintPubkey = stakePool.account.data.poolMint;
-      scnSolToken = new Token(
-        connection,
-        scnSolMintPubkey,
-        TOKEN_PROGRAM_ID,
-        stakerKeypair,
-      );
-      scnSolAtaPubkey = await Token.getAssociatedTokenAddress(
-        ASSOCIATED_TOKEN_PROGRAM_ID,
-        TOKEN_PROGRAM_ID,
-        scnSolMintPubkey,
-        staker.publicKey,
-      );
-    });
+  it("it calcSolDeposit() and calcSolDepositsInverse() works correctly", async () => {
+    const socean = new Socean("testnet");
+    const stakePool = await socean.getStakePoolAccount();
 
-    it("it deposits and withdraws on testnet", async () => {
-      const socean = new Socean("testnet");
-
-      // deposit 0.5 sol
-      const depositAmountSol = 0.5;
-      const depositAmount = depositAmountSol * LAMPORTS_PER_SOL;
-      console.log("deposit amount:", depositAmount);
-      const lastDepositTxId = (
-        await socean.depositSol(staker, new Numberu64(depositAmount))
-      )
-        .pop()
-        ?.pop();
-      // wait until the last tx (deposit) is finalized
-      await connection.confirmTransaction(lastDepositTxId!, "finalized");
-      console.log("deposit tx id:", lastDepositTxId);
-
-      // assert the balance decreased by ~0.5
-      const afterDepositBalanceLamports = await connection.getBalance(
-        staker.publicKey,
-        "finalized",
-      );
-      console.log("balance after deposit:", afterDepositBalanceLamports);
-      expect(afterDepositBalanceLamports).to.be.below(
-        originalBalanceLamports - depositAmountSol * LAMPORTS_PER_SOL,
-      );
+    const depositAmountSol = Math.random();
+    const depositAmount = Math.round(depositAmountSol * LAMPORTS_PER_SOL);
```

Comment:
think we should probably expand our test range instead of limiting it to 1 SOL
```suggestion
    const depositAmount = Math.round(Math.random() * Number.MAX_SAFE_INTEGER);
```